src/ga4-tracking.js

```diff
@@ -122,6 +45,7 @@ const trackGA4PageViewEvent = (eventData, dataLayerVariables) => {
     }
 
     const dealerIdValue = dealerId();
+    const market = window.document.location.hostname.split('.').pop() || null;
```

Comment:
The default values for DataLayer variables is `undefined`.
```suggestion
    const market = window.document.location.hostname.split('.').pop() || undefined;
```